string.c

```diff
@@ -3602,24 +3602,29 @@ str_uplus(VALUE str)
 
 /*
  * call-seq:
- *   -string -> frozen_string
- *   dedup -> frozen_string
+ *   -self -> frozen_string
  *
- * Returns a frozen, possibly pre-existing copy of the string.
+ * Returns a frozen string containing the same content as +self+.
```

Comment:
```suggestion
 * Returns a frozen string equal to +self+.
```

"containing content" is a bit redundant no?